see/image_providers/test/dummy_test.py

```diff
@@ -0,0 +1,18 @@
+import unittest
```

Comment:
If the image is missing you'll get an error somewhere downstream, this is the premise of the dummy provider, it's dummy. It just returns the path you give it.